### PR TITLE
ci: update needs-response workflow to use official stale action

### PR DESCRIPTION
**Pull request by mcansh**
Signed-off-by: Logan McAnsh <logan@mcan.sh>

Closes: #

- [ ] Docs
- [ ] Tests

Testing Strategy:

### CODE REVIEW

File: .github/workflows/no-response.yml [hunk 1]
```diff
@@ -7,20 +7,32 @@ on:
...
+
 jobs:
   noResponse:
     if: github.repository == 'remix-run/remix'
     runs-on: ubuntu-latest
 
     steps:
       - name: 🥺 Handle Ghosting
-        uses: lee-dohm/no-response@v0.5.0
+        uses: actions/stale@v5.0.0
```
Comment (MichaelDeBoey) [inline .github/workflows/no-response.yml:21]: You can just use latest version by doing

```suggestion
        uses: actions/stale@v5
```

File: .github/workflows/no-response.yml [hunk 2]
```diff
@@ -7,20 +7,32 @@ on:
...
+            This PR has been automatically closed because we haven't received a
+            response from the original author 🙈. This automation helps keep the issue
+            tracker clean from PRs that are unactionable. Please reach out if you
+            have more information for us! 🙂
+          days-before-close: 10
+          # don't automatically mark issues/PRs as stale
+          days-before-stale: -1
+          any-of-labels: needs-response
+          labels-to-remove-when-unstale: needs-response
+          repo-token: ${{ github.token }}
```
Comment (MichaelDeBoey) [inline .github/workflows/no-response.yml:38]: This is the default value, so no need to pass it

```suggestion
```

**Review by MichaelDeBoey — CHANGES_REQUESTED** [review]

File: .github/workflows/no-response.yml (hunk 1 above)
Comment (mcansh) [reply .github/workflows/no-response.yml:21]: 👍 copy pasta 🤪

**Review by kentcdodds — APPROVED** [review]
Looks good to merge. Why is this still a draft?

**Comment by mcansh** [comment]
> Looks good to merge. Why is this still a draft?

oops!

**Comment by MichaelDeBoey** [comment]
@mcansh We probably want to have this in `main` as well?

**Comment by kentcdodds** [comment]
I think we can wait on this one. No rush. Thanks for the reminder though @MichaelDeBoey!

**Pull request merged**